.github/workflows/test_release_linux.yml

```diff
@@ -0,0 +1,46 @@
+name: Logging Test Linux pipeline
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!master'
+env:
+  TAG: 1.7.4
+  GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
+  GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
+  AWS_S3_BUCKET_NAME: "nr-downloads-logging-testing"
```

Comment:
should point to staging, ref: https://github.com/newrelic/infrastructure-agent/blob/master/.github/workflows/prerelease_linux.yml#L21-L22